src/Nethermind/Nethermind.Serialization.Rlp/HeaderDecoder.cs

```diff
@@ -156,6 +162,11 @@ public class HeaderDecoder : IRlpValueDecoder<BlockHeader>, IRlpStreamDecoder<Bl
                 blockHeader.BaseFeePerGas = rlpStream.DecodeUInt256();
             }
 
+            if (blockHeader.Timestamp >= Eip4844TransitionTimestamp)
```

Comment:
quick question, do we have the spec for both Withdrawals + 4844? we're adding field in withdrawals too

Reply:
Yep, the current PR waits for @smartprogrammer93 changes